app/models/resource.rb

```diff
@@ -0,0 +1,10 @@
+class Resource < ActiveRecord::Base
+  belongs_to :conference
+  validate :used_less_than_quantity
+
+  private
+
+  def used_less_than_quantity
+    errors.add(:used, 'cannot be less than total quantity') unless used <= quantity
```

Comment:
The error message is wrong, it should be _can not be *higher* than total quantity_ :wink: